orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -176,6 +176,40 @@ def test_argmax(self):
         assert torch.allclose(cpu_result, ort_result.cpu())
         assert cpu_result.dim() == ort_result.dim()
 
+    def test_abs(self):
+        device = self.get_device()
+        cpu_tensor = torch.tensor([-1, -2, 3, -6, -7])
+        ort_tensor = cpu_tensor.to(device)
+
+        cpu_result = torch.abs(cpu_tensor)
+        ort_result = torch.abs(ort_tensor)
+
+        assert torch.allclose(cpu_result, ort_result.cpu())
```

Comment:
consider adding torch.equal check as well. not sure which is better equal or all close.

Reply:
Updated to equal - these are all ints, so I think equal is better - thanks!